lms/static/js/toggle_login_modal.js

```diff
@@ -86,14 +86,6 @@
 
                     $(modal_id).show().fadeTo(200, 1);
                     $(modal_id).find('.notice').hide().html('');
-                    var notice = $(this).data('notice');
-                    if (notice !== undefined) {
-                        $notice = $(modal_id).find('.notice');
-                        $notice.show().html(notice);
-            // This is for activating leanModal links that were in the notice. We should have a cleaner way of
-            // allowing all dynamically added leanmodal links to work.
-                        $notice.find('a[rel*=leanModal]').leanModal({top: 120, overlay: 1, closeButton: '.close-modal', position: 'absolute'});
-                    }
```

Comment:
This is dead code that was added in 2012 for https://github.com/edx/edx-platform/commit/61105134aaeb893584ea3c630eadf49eefc990ec#diff-26816f94ef5f3fecdb02bf5ea22c4ad4 and no longer seems to be used (I searched the repo for `data-notice` and `dataNotice` and found nothing. I'd prefer not to have to update it to be consistent with other changes I've made to this file if this is not being used.

Reply:
👍 To deleting dead code.